### PR TITLE
Add: Loading screen with Fake UAV intro

### PR DESCRIPTION
**Pull request by Vdauphin**
- [x] Move introshot to init.sqf file
- [x] Tanoa ?

Final test :
- [x] local
- [x] server

### CODE REVIEW

File: =BTC=co@30_Hearts_and_Minds.Altis/init.sqf [hunk 1]
```diff
@@ -1,6 +1,9 @@
 enableSaving [false,false];
 //Server
 call compile preprocessFile "core\fnc\compile.sqf";
+
+if (!isDedicated) then {btc_intro_done = call btc_fnc_intro;};
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/init.sqf]: if (hasInterface) then {
	//run on all player clients incl. player host
};

**Comment by Breech99** [comment]
In init_player line #29 shouldn't it be   ["5","nil","player"] remoteExec ["bis_fnc_int_ask_var",0];

remove the bis mp code. it's old and not used anymore.

File: =BTC=co@30_Hearts_and_Minds.Altis/init.sqf (hunk 1 above)
Comment (Breech99) [inline =BTC=co@30_Hearts_and_Minds.Altis/init.sqf]: what about ln 20-26 

if (!isDedicated && hasInterface) then {
	call compile preprocessFile "core\init_player.sqf";
};

if (!isDedicated && !hasInterface) then {
	call compile preprocessFile "core\init_headless.sqf";
};

Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/init.sqf]: has i know it is : https://github.com/Vdauphin/HeartsAndMinds/blob/master_daily/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/init.sqf#L13 https://github.com/Vdauphin/HeartsAndMinds/blob/master_daily/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/init.sqf#L17

**Comment by Vdauphin** [comment]
Should be but it is off topic here.

**Pull request merged**